### PR TITLE
Added using ISO details.

### PR DESCRIPTION
**Pull request by tonybaltovski**

### CODE REVIEW

File: docs_versioned_docs/version-ros2humble/ros/installation/robot.mdx [hunk 1]
```diff
@@ -12,12 +12,54 @@ import ComputerNetworkingSetupInstall from "../../components/networking/_compute
...
+Though other operating systems are supported, it is highly recommended to use Ubuntu 22.04. 
+The preferred method is using the Clearpath Robotics ISO image, which is covered in this section.
+
+:::note
+
+The Clearpath Robotics ISO image only targets x86 computers (`amd64` architecture).
+
+:::
+
+Clearpath provides a lightly customized installation image of Ubuntu 22.04 that automatically has the needed script to setup the robot.
```
Comment (jhiggins-cpr) [inline docs_versioned_docs/version-ros2humble/ros/installation/robot.mdx]: I think we can remove "automatically" here.

File: docs_versioned_docs/version-ros2humble/ros/installation/robot.mdx [hunk 2]
```diff
@@ -12,12 +12,54 @@ import ComputerNetworkingSetupInstall from "../../components/networking/_compute
...
+To install the software on a physical robot through the Clearpath Robotics ISO image, you will first need a USB drive of at least 4 GB
+to create the installation media, an ethernet cable, a monitor, and a keyboard.
+
+1.  Download the [ROS 2 Humble ISO image](https://packages.clearpathrobotics.com/stable/images/1.0.0/).
```
Comment (jhiggins-cpr) [inline docs_versioned_docs/version-ros2humble/ros/installation/robot.mdx:29]: Will we update this link each time there is a new release?

**Review by jhiggins-cpr — COMMENTED** [review]
Looks good. Just a couple very minor items.

File: docs_versioned_docs/version-ros2humble/ros/installation/robot.mdx (hunk 2 above)
Comment (tonybaltovski) [reply docs_versioned_docs/version-ros2humble/ros/installation/robot.mdx:29]: That is correct.

**Review by jhiggins-cpr — APPROVED** [review]

**Pull request merged**